packages/navigation/src/navigation.js

```diff
@@ -200,9 +200,10 @@ export default class Navigation {
 	 * Adds JS classes and initial AIRA attributes.
 	 */
 	setupSubMenus() {
+		const id = this.$menu.getAttribute('id');
```

Comment:
I'm not sure if `id` is mandatory but regardless it would be a good thing to fallback to an empty string in case there's no id.

`const id = this.$menu.getAttribute('id') ?? ''`;